qiskit_ibm_provider/utils/backend_converter.py

```diff
@@ -107,7 +108,7 @@ def convert_to_target(
         target.add_instruction(Measure())
     # parse global configuration properties
     if hasattr(configuration, "dt"):
-        target.dt = configuration.dt
+        target.dt = configuration.dt * 1e-9
```

Comment:
Same this is already done here https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/providers/models/backendconfiguration.py#L355

Reply:
```suggestion
        target.dt = configuration.dt
```